### DISCUSSION TITLE
Nuxt3 + Jest, ReferenceErrors

### DISCUSSION THREAD

**Discussion by jkneepkens** [nuxt/nuxt#17805] [Questions] [answered]
Hi there, i'm trying to use Jest (@vue/test-utils) in combination with Nuxt3, but i'm getting these Reference Errors when I run some of my tests, for example:

- ReferenceError: ref is not defined
- ReferenceError: useFetch is not defined

I know this is because for Nuxt3, we don't need to import these in the Vue files itself, because they are auto-imported by Nuxt, but Jest / Vue Test Utils doesn't understand / have this logic.

Does anyone know how to fix this or having a good workaround for it? We don't want to update the Vue files by adding al those imports, because then we loose the *magic* of Nuxt of course! Thanks in advance!

**Comment by danielroe** [comment, marked as answer]
This is the issue to follow: https://github.com/nuxt/framework/issues/2465
